### PR TITLE
docs: Update Gauges adding why, how with data reqs and images

### PR DESCRIPTION
**Pull request by srperf**
**What is this feature?**

This PR adds some elements to the gauge doc:
- Why, explaining why and examples when one could need to use it.
- Structuring the configurations section with video and play elements within
- Add how to descriptions with data needs
- Add several images describing the gauge data functionalities

**Why do we need this feature?**

This will help users understand the gauge visualization better

**Who is this feature for?**

Any user who needs to create a gauge panel

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/gauge/index.md [hunk 1]
```diff
@@ -25,16 +25,93 @@ refs:
 
 # Gauge
 
-Gauges are single-value visualizations that can repeat a gauge for every series, column or row.
+Gauges are single-value visualizations that allow you to show where a value falls within a defined range. You can use the visualization to display one or more gauges, each corresponding to a different series, column, or row, with the values bound by minimum and maximum thresholds.
```
Comment (drew08t) [inline docs/sources/panels-visualizations/visualizations/gauge/index.md:28]:
```suggestion
Gauges are single-value visualizations that allow you to quickly visualize where a value falls within a defined or calculated min and max range. With repeat options, you can display multiple gauges, each corresponding to a different series, column, or row.
```

File: docs/sources/panels-visualizations/visualizations/gauge/index.md [hunk 2]
```diff
@@ -25,16 +25,93 @@ refs:
...
 
-{{< docs/play title="Grafana Gauge Visualization" url="https://play.grafana.org/d/KIhkVD6Gk/" >}}
+You can use gauges if you need to track:
+
+- Service level objectives (SLOs)
+- How full your gas tank is
+- How fast a vehicle is moving inside the engine speeds limits
+- Body temperature
+- CPU consumption (0-100%)
+- RAM availability
```
Comment (drew08t) [inline docs/sources/panels-visualizations/visualizations/gauge/index.md:39]:
```suggestion
- Service level objectives (SLOs)
- How full a piece of equipment is
- How fast a vehicle is moving within a set of limits
- Network latency
- Equipment state with setpoint and alarm thresholds
- CPU consumption (0-100%)
- RAM availability
```

File: docs/sources/panels-visualizations/visualizations/gauge/index.md [hunk 3]
```diff
@@ -135,6 +212,10 @@ Adjust the sizes of the gauge text.
 
 {{< docs/shared lookup="visualizations/thresholds-options-2.md" source="grafana" version="<GRAFANA_VERSION>" >}}
 
+Last, gauge colors and thresholds (the lines outside the circles) of the gauge can be configured as described above.
```
Comment (drew08t) [inline docs/sources/panels-visualizations/visualizations/gauge/index.md:215]:
```suggestion
Last, gauge colors and thresholds (the outer bar markers) of the gauge can be configured as described above.
```

Changed to use the same language as the options menu.

**Comment by srperf** [comment]
@imatwawana 
I think this one is ready, just check some of the list formatting updates.

@drew08t Thank you so much!

File: docs/sources/panels-visualizations/visualizations/gauge/index.md [hunk 4]
```diff
@@ -25,16 +25,17 @@ refs:
 
 # Gauge
 
-Gauges are single-value visualizations that allow you to show where a value falls within a defined range. You can use the visualization to display one or more gauges, each corresponding to a different series, column, or row, with the values bound by minimum and maximum thresholds.
+Gauges are single-value visualizations that allow you to quickly visualize where a value falls within a defined or calculated min and max range. With repeat options, you can display multiple gauges, each corresponding to a different series, column, or row.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/gauge/index.md:28]:
```suggestion
Gauges are single-value visualizations that allow you to quickly visualize where a value falls within a defined or calculated minimum to maximum range. With repeat options, you can display multiple gauges, each corresponding to a different series, column, or row.
```

**Comment by imatwawana** [comment]
hey @drew08t - thanks so much for the review. The only thing I'm concerned about is this phrase:

> With repeat options, you can display multiple gauges, each corresponding to a different series, column, or row.

We took something similar out of the original intro partly because I don't know what it means. There's nothing in the documentation to explain what's meant by "repeat options". What specifically do you mean?

**Comment by drew08t** [comment]
> hey @drew08t - thanks so much for the review. The only thing I'm concerned about is this phrase:
> 
> > With repeat options, you can display multiple gauges, each corresponding to a different series, column, or row.
> 
> We took something similar out of the original intro partly because I don't know what it means. There's nothing in the documentation to explain what's meant by "repeat options". What specifically do you mean?

Repeated options can be found under Panel options:
image

**Review by drew08t — APPROVED** [review]

**Comment by imatwawana** [comment]
Ahhhh okay. I think we should explicitly say this vs alluding to it. I was under the impression it was a special feature specific to gauges. For the sake of not letting this PR go stale, let's go ahead with how it's written now and I'll open an issue to clarify that in the docs later.

**Pull request merged**